### PR TITLE
fix Dockerfile to follow latest change and rearrange directory

### PR DESCRIPTION
**Pull request by suleisl2000**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/mxnet-operator/25)

### CODE REVIEW

File: manifests/deployment.yaml [hunk 1]
```diff
@@ -1,22 +1,22 @@
-apiVersion: apps/v1
+apiVersion: extensions/v1beta1
```
Comment (gaocegege) [inline manifests/deployment.yaml]: I think there is no need to update here.

Comment (suleisl2000) [reply manifests/deployment.yaml]: Yes, I just make it same as pytorch. I will keep it if no issue is introduced. :)

**Pull request merged**